client/packages/common/src/index.ts

```diff
@@ -17,7 +17,6 @@ export {
   useQuery,
   UseQueryResult,
   QueryClient,
-  QueryClientProvider,
```

Comment:
This breaks plugins! not sure why it was added back in here.

Reply:
not yet, I think. Just needed to get some screenshots for a proposal. I'm wondering about creating the dashboard widgets solely as plugins and having them configurable and if we get to that point, the plugin framework is tested by everyone!